pkg/reconciler/serverlessservice/resources/services.go

```diff
@@ -59,8 +82,8 @@ func MakePublicService(sks *v1alpha1.ServerlessService) *corev1.Service {
 			Ports: []corev1.ServicePort{{
 				Name:       servicePortName(sks),
 				Protocol:   corev1.ProtocolTCP,
-				Port:       servicePort,
-				TargetPort: intstr.FromString(requestQueuePortName),
+				Port:       pubServicePort(sks),
+				TargetPort: pubTargetPort(sks, hasBackends),
```

Comment:
cc @tcnghia 

This means we will have to touch two resources, which is twice what I'd like.  Do we know what needs to happen to remove the need to change things here?  Can we figure that out and open an issue?

Reply:
Since CI/VS operate on services, we have to make them look ship shape. But it's the same reason as the other changes.

Reply:
Can you open an issue with what we discussed this morning (while it is fresh)?

Reply:
#3873 
Done